### DISCUSSION TITLE
There was a problem submitting the form

### DISCUSSION THREAD

**Discussion by forgecodex** [laravel/nova-issues#5319] [Q&A]
- Laravel Version: 10.0.0
- Nova Version: 4.22.0
- PHP Version: 8.2
- Database Driver & Version:Mysql8.0
- Operating System and Version: #.#
- Browser type and version: #.#
- Reproduction Repository: https://github.com/###/###

### Description:

Can not login display show "There was a problem submitting the form"

**Comment by crynobone** [comment]
You need to debug by inspecting `laravel.log`. One possibility is that the database connection is not properly configured for the project.
